tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -580,6 +580,8 @@ var _ = Describe("Post-Check testing for cluster deletion",
 				clusterDetail, err := profilehandler.ParserClusterDetail()
 				Expect(err).To(BeNil())
 				oidcEndpointUrlC = clusterDetail.OIDCEndpointURL
+				index := strings.Index(oidcEndpointUrlC, ".com")
+				oidcEndpointUrlC = oidcEndpointUrlC[:index+4]
```

Comment:
Let us use the package `net/url` to get the path.
```
u, err := url.Parse(oidcEndpointUrlC)
Expect(err).To(BeNil())
oidcEndpointUrl := u.String()

Reply:
fixed, Thanks for the suggestion!